drizzlepac/devutils/fix_json_files.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
```

Comment:
Does this file need to be provided to the end-user?  This seems to be a purely developmental function...